src/tests/components/CalcBtn.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import CalcBtn from '../../components/CalcBtn'; // eslint-disable-line
+
+test("Calculator's buttons display correctly", () => {
+  const onClick = jest.fn();
+  const tree = renderer.create(<CalcBtn onClick={onClick} />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] There is path issue that need to be resolved here. Kindly look into it and fix. 

![calculate-logic](https://user-images.githubusercontent.com/26597432/228854085-b43071d9-510f-4e0b-9060-c74fdf749cc8.png)